tests/nvmeof/test_ceph_nvmeof_neg_tests.py

```diff
@@ -958,58 +790,54 @@ def test_ceph_83581753(ceph_cluster, rbd, pool, config):
     Returns:
         int: 0 on success, 1 on failure.
     """
-
-    gw_node = get_node_by_id(ceph_cluster, config["gw_node"])
-    ceph = Orch(ceph_cluster, **{})
-    nvmegwcli = create_gateway(
-        nvme_gw_cli_version_adapter(ceph_cluster),
-        gw_node,
-        mtls=config.get("mtls"),
-        shell=getattr(ceph, "shell"),
-        port=config.get("gw_port", 5500),
-        gw_group=config.get("gw_group"),
-    )
-
-    subsystem = dict()
-    listener_port = find_free_port(gw_node)
-    subsystem.update(
-        {
-            "nqn": "nqn.2016-06.io.spdk:ceph-83581753",
-            "serial": 83581753,
-            "listener_port": listener_port,
-            "allow_host": "*",
-        }
-    )
-    initiator_cfg = {
-        "subnqn": "nqn.2016-06.io.spdk:ceph-83575814",
-        "listener_port": listener_port,
-        "node": config.get("initiator_node"),
-    }
     try:
-        configure_subsystems(ceph_cluster, rbd, pool, nvmegwcli, subsystem)
+        # Deploy NVMe Service
+        if config.get("install"):
+            nvme_service.deploy()
+
+        # Initialize gateways
+        nvme_service.init_gateways()
+        nvmegwcli = nvme_service.gateways[0]
+
+        # Configure Subsystems
+        configure_subsystems(nvme_service)
```

Comment:
Can we not directly call configure_gw_entities and that will take care of configuring subsystems, listeners and hosts right? All you need to do is pass those keywords to config I guess. Can we check this?